graphistry/compute/ai/symbolic.py

```diff
@@ -417,37 +411,50 @@ def __init__(self, index, all_indexes=False, verbose=False, *args, **kwargs):
         self.fields = []
         self.verbose = verbose
         self.all_indexes = all_indexes
-        self.mem = {}
-        self.antimem = {}
         # symbolic Splunk Symbol
         self.splunk = Splunk()
 
     def open_context(self):
         # load it once ...
+        import json
         if os.path.exists('splunk.context'):    
```

Comment:
@silkspace maybe make a session key? later this should be by connector or something

Reply:
(my thinking here wrt splunk.context is we're actively move to a multi-user / multi-connector setting, so a context shared across st users will leak info)